### PR TITLE
Update media/audio

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**

### CODE REVIEW

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/media/embed.md

**Comment by enki-hq** [comment]
Failed parsing files: 
  - Web/HTML/media/video.md: 
 > Must have exactly 1 yaml configuration but found 0 instead.

**Comment by enki-hq** [comment]
Failed parsing files: 
  - Web/HTML/media/video.md: 
 > Must have exactly 1 yaml configuration but found 0 instead.

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/media/video.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/media/video.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/computer-code/intro-code.md

**Comment by enki-hq** [comment]
Rule insight:quiz failed for files: 
  - Web/HTML/computer-code/intro-code.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - Web/HTML/computer-code/intro-samp.md

**Comment by enki-hq** [comment]
Rule insight:code-width failed for files: 
  - Web/HTML/computer-code/intro-var.md

File: Web/HTML/computer-code/intro-var.md [hunk 1]
```diff
@@ -11,29 +11,65 @@ category: must-know
...
+var {
+  font: bold 20px "Times New Roman",
+  Times, serif;
+}
+```
 
 ---
 ## Practice
 
 When using the `<var>` element, which statements are true?
```
Comment (lizTheDeveloper) [inline Web/HTML/computer-code/intro-var.md]: If the user selects the correct elements but in the wrong order, they will still get it wrong. 
We don't have a proper control for "select all elements that are true". Normally when there are multiple true statements, what you want is to pick the one that's false:
"When using the <var> element, which statement is false?"

- false statement
- true statement
- true statement
- true statement

File: Web/HTML/computer-code/intro-code.md [hunk 2]
```diff
@@ -11,21 +11,48 @@ category: must-know
...
+The default font style for the `<code>` element for most browsers is the monospace font.
+ 
+**Note: It's best to use the `<code>` element to style content for HTML elements, CSS rules, filenames and JS scripts.**
+ 
 ---
 ## Practice
 
 What content is best to style when using the `<code>` element?
 
+???
```
Comment (lizTheDeveloper) [inline Web/HTML/computer-code/intro-code.md]: Gotta change all these questions, where there are multiple true answers that could be improperly ordered

**Comment by lizTheDeveloper** [comment]
This has been migrated to #766

**Pull request closed without merging**